pkg/minishift/addon/addon_meta.go

```diff
@@ -95,6 +96,10 @@ func (meta *DefaultAddOnMeta) VarDefaults() ([]RequiredVar, error) {
 		defaults := make([]RequiredVar, 0, len(items))
 		for _, item := range items {
 			varDefault, _ := minishiftStrings.SplitAndTrim(item, "=")
+			// In case of value of varDefault is null/Null/NULL then value converted to empty string.
+			if strings.ToLower(varDefault[1]) == "null" {
+				varDefault[1] = ""
+			}
```

Comment:
https://github.com/minishift/minishift/issues/2464#issuecomment-400266767

Reply:
not sure if I like to see `null`... or rather `nil`.
nil has a clearer meaning in language than `null`, as the last one means related to zero or invalid, while nil means non-existant.

does it actually NEED a value? why is `FOO=` not possible?

Reply:
> nil has a clearer meaning in language than null, as the last one means related to zero or invalid, while nil means non-existant.

@gbraad sure that something we can modify

> does it actually NEED a value? why is FOO= not possible?

@gbraad https://github.com/minishift/minishift/issues/2464#issuecomment-400266767

Reply:
Both `nil` or `null` works for me.